protocol/src/test/java/net/md_5/bungee/protocol/ProtocolTest.java

```diff
@@ -0,0 +1,24 @@
+package net.md_5.bungee.protocol;
+
+import net.md_5.bungee.protocol.packet.PlayerListHeaderFooter;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class ProtocolTest
+{
+
+    @Test
+    public void testHeaderFooter() throws Exception
+    {
+        Protocol.DirectionData protocol = Protocol.GAME.TO_CLIENT;
```

Comment:
While it's far from fantastic it does at least verify that the correct ids are returned. If this was just a simple enum I would agree its entirely useless but with the linked protocol logic I think there is some benefit to testing it. 

All that being said if you would prefer more than happy to remove this test from the PR